package.json

```diff
@@ -27,6 +27,12 @@
   "engines": {
     "node": ">=4.2.2"
   },
+  "dependencies": {
```

Comment:
On line 7, it says the bin's name should be `service-runner`. Obviously, it needs `s/runner/builder`